openmrs-19/src/test/java/org/motechproject/openmrs19/it/MRSPersonServiceIT.java

```diff
@@ -61,15 +61,20 @@ public void initialize() throws InterruptedException {
     public void shouldCreatePerson() {
 
         assertNotNull(person.getId());
-        assertEquals(person.getPersonId(), mrsListener.eventParameters.get(EventKeys.PERSON_ID));
-        assertEquals(person.getFirstName(), mrsListener.eventParameters.get(EventKeys.PERSON_FIRST_NAME));
-        assertEquals(person.getLastName(), mrsListener.eventParameters.get(EventKeys.PERSON_LAST_NAME));
+        assertEquals(mrsListener.eventParameters.get(EventKeys.PERSON_ID), person.getPersonId());
+        assertEquals(mrsListener.eventParameters.get(EventKeys.PERSON_FIRST_NAME), person.getFirstName());
+        assertEquals(mrsListener.eventParameters.get(EventKeys.PERSON_LAST_NAME), person.getLastName());
 
         assertTrue(mrsListener.created);
         assertFalse(mrsListener.deleted);
         assertFalse(mrsListener.updated);
     }
 
+    /*
```

Comment:
add comment about address above line with assertTrue connected with address. Do not add javadoc to test method.

Moreover this comment is unclear. I suggest the following :
'So far OpenMRS module stores only one field of person's address, which is 'address1'. However while retrieving person from OpenMRS server all person's address fields are put into one string. That's why it is checked if address field contains address1 value.'